operations/fetch_test.go

```diff
@@ -1,84 +0,0 @@
-package operations
```

Comment:
Aren't the tests still relevant?

Reply:
The tests were added by the commit I am reverting, so I was guessing not.

Reply:
The tests were added because there were no tests before. The logical tests are still relevant but implementation of them will be different